clients/src/main/resources/common/message/FetchResponse.json

```diff
@@ -53,9 +53,9 @@
       "about": "The response topics.", "fields": [
       { "name": "Topic", "type": "string", "versions": "0+", "entityType": "topicName",
         "about": "The topic name." },
-      { "name": "PartitionResponses", "type": "[]FetchablePartitionResponse", "versions": "0+",
+      { "name": "Partitions", "type": "[]PartitionData", "versions": "0+",
         "about": "The topic partitions.", "fields": [
-        { "name": "Partition", "type": "int32", "versions": "0+",
+        { "name": "Index", "type": "int32", "versions": "0+",
```

Comment:
We typically call this `PartitionIndex` in other requests/responses. Is that right?

Reply:
> We typically call this PartitionIndex in other requests/responses. Is that right?

You are right!